tests/trello_tasks_backend.test.js

```diff
@@ -0,0 +1,137 @@
+const TrelloTasksBackend = require('../src/trello_tasks_backend')
+const nock = require('nock')
+
+describe('Test handling requests to and from Trello', () => {
+  test('Successfully get and translate tasks from Trello', async () => {
+    const tasksBackend = new TrelloTasksBackend()
+    const apiBaseUrl = 'https://api.trello.com/1'
+    const listId = '5'
+    const apiKey = 'yeKipa'
+    const apiToken = 'nekpoTipa'
+    const config = { apiBaseUrl, listId, apiKey, apiToken }
+
+    const scope = nock('https://api.trello.com')
+      .get(`/1/lists/${listId}/cards?key=${apiKey}&token=${apiToken}`)
+      .reply(200, [
+        {
+          id: '5cf705cfe9bb5d1d62d03375',
+          checkItemStates: null,
+          closed: false,
+          dateLastActivity: '2020-01-30T00:21:30.911Z',
+          desc: 'description of card',
+          descData: null,
+          dueReminder: null,
+          idBoard: '3edf95f8d06c1bbb2b638886',
+          idList: '028b261023807ede7dbb9de8',
+          idMembersVoted: [],
+          idShort: 2,
+          idAttachmentCover: null,
+          idLabels: [],
+          manualCoverAttachment: false,
+          name: 'Test Card',
+          pos: 98303.5,
+          shortLink: 'f01ad573',
+          isTemplate: false,
+          badges: {
+            attachmentsByType: {
+              trello: {
+                board: 0,
+                card: 0
+              }
+            },
+            location: false,
+            votes: 0,
+            viewingMemberVoted: false,
+            subscribed: true,
+            dueComplete: false,
+            due: null,
+            description: false,
+            attachments: 0,
+            comments: 1,
+            checkItemsChecked: 0,
+            checkItems: 0,
+            fogbugz: '',
+            checkItemsEarliestDue: null
+          },
+          dueComplete: false,
+          due: null,
+          idChecklists: [],
+          idMembers: [],
+          labels: [],
+          shortUrl: 'https://trello.com/c/f01ad573',
+          subscribed: true,
+          url: 'https://trello.com/c/f01ad573/testboardname',
+          cover: {
+            idAttachment: null,
+            color: null,
+            idUploadedBackground: null,
+            size: 'normal',
+            brightness: 'light'
+          }
+        }
```

Comment:
Do we want to move this to its own JSON file for readability?

Reply:
Good call. Any idea how to get the ts-jsdocs to be cool with JSON imports? (see [line 4 of `trello_tasks_backend.test.js`](https://github.com/codeforlansing/cityzen-server/pull/35/files#diff-996b6219beebcf0d546926ba10bfdd2bR4))